test/verify/check-system-journal

```diff
@@ -471,18 +471,18 @@ ExecStart=/bin/sh -c 'sleep 5; for s in $(seq 10); do echo SLOW; sleep 0.1; done
 
         m.execute(r"printf 'MESSAGE=Hello \01 World\nPRIORITY=3\nFOO=bar\nBIN=a\01b\02c\03\n' | logger --journald")
         self.login_and_go("/system/logs#/?tag=logger&since=@" + self.test_start_time)
-        b.click("#journal-box .cockpit-logline .cockpit-log-message:contains('[13 bytes of binary data]')")
-        b.wait_text(".pf-c-card__title", "[13 bytes of binary data]")
+        b.click("#journal-box .cockpit-logline .cockpit-log-message:contains('Hello \01 World')")
+        b.wait_text(".pf-c-card__title", "Hello \01 World")
 
         def wait_field(name, value):
             row_sel = "dt:contains('" + name + "')"
             b.wait_visible(row_sel + "+ dd:contains('" + value + "')")
 
         wait_field("FOO", "bar")
-        wait_field("BIN", "[6 bytes of binary data]")
+        wait_field("BIN", "a\01b\02c\03")
```

Comment:
That's the wrong case. Binary data is used for core dumps, trec/session-recording, perhaps screenshots etc, and it would just overwhelm the page, the browser, and the user to dump megabytes or even gigabytes of binary data here.

Reply:
i see

Reply:
The BIN field shouldn't be decoded. I don't see where that happens in the code at first sight, so this needs a more thorough debug session.